internal/helpers/rego/config/orbs.rego

```diff
@@ -16,4 +16,11 @@ ban_orbs_version(banned_orbs) = { orb: msg | orb := banned_orbs[_]
   [name, version] := split(orb, "@")
   orbs[name] == version
   msg := sprintf("%s orb is not allowed in CircleCI configuration", [orb])
-}
\ No newline at end of file
+}
+
+orbs_allowlist(allowed_orbs) = { orb: msg | orb := input["orbs"][_]
```

Comment:
I am not sure this helper is accomplishing what we said in the slack thread.

As far as I can tell it:
- has a guard to check if there are orbs, otherwise it early exits
- Looks for the orb value name but not key:
```yaml
orbs:
  key: name@version
```
- then if the name is not in allowed list of names it returns an error

This isn't very secure because all it does is allow a list of orbs but it doesn't specify how they must be namespaced "keyed" nor does it accept multiple versions of an orb.

What is preferable from a customer perspective?
```
config.orbs_allowlist({"good", "safe", "ok"})
```
Such that you can only import orbs with those names?

OR

Police the keys to fully qualified orb sets?

```
config.orb_allowlist({
  "good": "good@1.0.0",
  "ok": {"ns/ok@1.5.0", "ns/ok@2.0.0",
})
```

Reply:
I explicity left off version,  but disagree on the name/key issue, and have a test to measure that explicity.

Orbs are targeted by `namespace/name:version` and this rule makes sure `namespace\name` is allowed, regardless of the local key they assign it.

Any orbs defined with  a namespace./ name not allowed will fail.

Reply:
We dont care what devs call it within their config, as long as we approve its use.

Reply:
Ok... I have just found that often we want to make sure that orbs are assigned to a known key so that we can enforce their execution within steps. Although we can decide that it is best left to the specific policy that enforces certain commands to enforce the orb key to ns+name+version...

Its a locality versus centralization problem